MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -54,17 +54,18 @@ uint32_t GPS_coord_to_degrees(char* s) {
     min += DIGIT_TO_VAL(*q++);
   }
   // convert fractional minutes
-  // expect up to four digits, result is in
+  // expect up to four (five) digits, result is in
   // ten-thousandths of a minute
   if (*p == '.') {
     q = p + 1;
-    for (i = 0; i < 4; i++) {
+    for (i = 0; i < 5; i++) {
       frac_min *= 10;
```

Comment:
This is you bug too - you change loop count but don't change factor so fractional part will be multiplied by 10